subworkflows/local/profiling.nf

```diff
@@ -114,6 +123,10 @@ workflow PROFILING {
         ch_versions = ch_versions.mix( METAPHLAN3.out.versions.first() )
     }
 
+    if ( params.run_kaiju ) {
+        KAIJU_KAIJU ( ch_input_for_kaiju.reads, ch_input_for_kaiju.db )
```

Comment:
Looks good for now ,but once we have the kaju2table module (should add that next actually!)

then we can add to MultiQC! https://multiqc.info/docs/#kaiju

Reply:
I am resolving this.